docs/UPDATING_DEPENDENCIES.md

```diff
@@ -0,0 +1,120 @@
+# Updating @elastic/synthetics and Playwright Versions
+
+This guide explains how to update the `@elastic/synthetics` package and its associated Playwright dependencies in the synthetics-recorder project.
+
+## Background
+
+This project uses a custom fork of Playwright maintained at [elastic/playwright](https://github.com/elastic/playwright). The fork includes additional patches required for the recorder functionality. When updating `@elastic/synthetics`, you must also update the Playwright packages to matching versions.
+
+## Prerequisites
+
+- Access to the [elastic/playwright](https://github.com/elastic/playwright) repository
+- Access to the [elastic/synthetics](https://github.com/elastic/synthetics) repository
+- Git configured with appropriate permissions to push to elastic/playwright
+
+## Update Process
+
+> **Note:** The examples in this guide illustrate upgrading Playwright from version `1.56.1` to `1.57.0`. Replace these versions with your actual current and target versions.
+
+### Step 1: Identify the Required Playwright Version
+
+Check which Playwright version is used by the target `@elastic/synthetics` version:
+
+1. Go to the [elastic/synthetics](https://github.com/elastic/synthetics) repository
+2. Find the version you want to update to (e.g., `1.22.0`)
+3. Check the `package.json` to identify the `playwright`, `playwright-core`, and `playwright-chromium` versions
+
+### Step 2: Create the New Playwright Branch
+
+In the [elastic/playwright](https://github.com/elastic/playwright) fork:
+
+1. Checkout the corresponding Microsoft Playwright release tag
+
+   ```bash
+   git fetch upstream
+   git checkout v1.57.0  # Replace with target version
```

Comment:
Might be better to display this as env var kind of syntax to make it more explicit that you need to customize the command. Also removes the need for the comment.
```suggestion
   git checkout ${TARGET_VERSION}
```

Reply:
Makes sense!

https://github.com/elastic/synthetics-recorder/pull/576/commits/d336e3048e4a268be40d9802714f5fd7493b28aa